src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/utilities/Configs.scala

```diff
@@ -122,8 +122,11 @@ object JobDefinitionConfig {
 object HealthConfig {
   lazy val databaseConnectionTimeoutMillis: Int =
     Try(Configs.conf.getInt("health.databaseConnection.timeoutMillis")).getOrElse(120000)
-  lazy val yarnConnectionTestEndpoint: String =
+  lazy val yarnConnectionTestEndpoint: String = {
     Configs.conf.getString("health.yarnConnection.testEndpoint")
+  }
+  lazy val yarnConnectionTimeoutMillisOpt: Option[Int] =
```

Comment:
What about no option with default value like in databaseConnectionTimeoutMillis

Reply:
You don't have to set a connection timeout on the `URLConnection`. If I provide a default value, you cannot configure that behaviour anymore.

Reply:
But why the different behavior for yarn compared to DB?